src/scope-hoisting/mangler.js

```diff
@@ -0,0 +1,69 @@
+const rename = require('./renamer');
+const t = require('babel-types');
+
+const CHARSET = (
+  'abcdefghijklmnopqrstuvwxyz' + 'ABCDEFGHIJKLMNOPQRSTUVWXYZ$_'
+).split('');
+
+/**
+ * This is a very specialized mangler designer to mangle only names in the top-level scope.
+ * Mangling of names in other scopes happens at a file level inside workers, but we can't
+ * mangle the top-level scope until scope hoisting is complete in the packager.
+ */
+function mangleScope(scope) {
```

Comment:
I can see babel minify license being violated here, Parcel is using the [babel minify's mangler](https://github.com/babel/minify/blob/master/packages/babel-plugin-minify-mangle-names/src/charset.js)code which is licensed(which is MIT). Can you please mention link to the project and the license?